migrations/redis/01-create-constants/index.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Maybe we should add [no-multiple-empty-lines](https://eslint.org/docs/rules/no-multiple-empty-lines) rule, with `maxBOF`: 0?